PixaPicker/ViewController.swift

```diff
@@ -7,16 +7,33 @@
 //
 
 import UIKit
-
 import SDWebImage
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, UISearchResultsUpdating, UISearchBarDelegate {
+    
+    let apiKey = "key=13683470-874d69ddffa828cbb82551e32" //API key for my Pixabay account
     @IBOutlet weak var picsCollection: UICollectionView!
-    @IBOutlet weak var searchBar: UISearchBar!
+    
+    func updateSearchResults(for searchController: UISearchController) {
+        let searchText = searchController.searchBar.text!
+        let searchTextWithoutSpaces = searchText.replacingOccurrences(of: " ", with: "+")
+        let urlWithAPI = "https://pixabay.com/api/?" + apiKey
+        let urlWithSeachText = urlWithAPI + "&q=" + searchTextWithoutSpaces
+        let finalURL = urlWithSeachText + "&image_type=photo"
+        print(finalURL) //FOR TESTING ONLY
+    }
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        print("TEST")
+        let searchController = UISearchController(searchResultsController: nil)
+        searchController.searchResultsUpdater = self
+        searchController.searchBar.delegate = self
+        searchController.obscuresBackgroundDuringPresentation = false
+        searchController.searchBar.placeholder = "Image Search"
+        navigationItem.searchController = searchController
+        searchController.searchBar.sizeToFit()
+        navigationItem.hidesSearchBarWhenScrolling = false
```

Comment:
I'd move this logic to a `private func setupSearchController()` then call that from viewdidload